src/java/org/apache/cassandra/index/sasi/conf/DataTracker.java

```diff
@@ -77,6 +77,11 @@ public View getView()
         }
         while (!view.compareAndSet(currentView, newView));
 
+        for(SSTableReader sstable: indexedSSTables)
```

Comment:
Nit: missed whitespaces:
```suggestion
        for (SSTableReader sstable : indexedSSTables)
```